portfolio/src/main/webapp/comments.html

```diff
@@ -25,23 +25,43 @@
           <a class="nav-item nav-link active" href="#">Comments<span class="sr-only">(current)</span></a>
           </div>
         </div>
+        <!-- Hidden button that triggers modal, will show if user is not logged in. -->
+        <button type="button" class="btn btn-light" id="modal-btn" data-toggle="modal" data-target="#login-modal">
+          Login!
+        </button>
+        <!-- Hidden button to logout, will show if user is logged in. -->
+        <button type="button" class="btn btn-light" id="logout-btn">
+          Logout!
+        </button>
       </nav>
     </div>
     <div id="content">
       <div id="title">
         <h1>Comments</h1>
       </div>
-      <div class="container">
-        <div class="row">
-          <div class="col-4">
-            <button class="btn btn-outline-light" onclick="checkLoginStatus()">Login Status!</button>
-          </div>
-          <div class="col-4">
-            <p id="status-label"></p>
+      <!-- Hidden modal that will appear if login button is clicked. -->
+      <div class="modal fade" id="login-modal" tabindex="-1" role="dialog" 
+          aria-labelledby="exampleModalLabel" aria-hidden="true">
+        <div class="modal-dialog">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title" id="exampleModalLabel">Login</h5>
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+            <div class="modal-body">
+              <p>Login to submit comments!</p>
```

Comment:
nit: 'Login' is a noun, and 'log in' is a verb! So should be 'Log in' here!

Reply:
Ack!